apps/docs/src/app/core/component-docs/bar/examples/bar-default-example.component.html

```diff
@@ -1,6 +1,6 @@
 <div fd-bar>
     <div fd-bar-left>
-        <fd-button-bar [glyph]="navigationArrow$ | async"></fd-button-bar>
+        <fd-button-bar ariaLabel='back' [glyph]="navigationArrow$ | async"></fd-button-bar>
```

Comment:
Please be consistent with the `'` and `"`. Also, please provide a more meaningful label than 'back' and 'Nature'.

Reply:
modified.